src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,83 @@
+package duke.storage;
+
+import java.util.ArrayList;
+
+import duke.commons.exceptions.DukeException;
+import duke.model.*;
+import duke.model.user.User;
+import duke.logic.autocorrect.Autocorrect;
+
+import static duke.commons.FilePaths.*;
+
+/**
+ * Storage is a public class, a storage class encapsulates the filePath to read from disk and write to disk.
+ * @author Ivan Andika Lie
+ */
+public class Storage {
+    private Load loader = new Load();
+    private Write writer = new Write();
+
+    /**
+     * This is a function that will load all info required to initialize a MealList object.
+     * @author Chua Zong Wei
+     */
+    public void load(MealList meals) throws DukeException {
+        loader.loadFile(meals, DATA_FILE);
```

Comment:
in this particular case, you may want to qualify your constant names to make it easy to find
(or import them individually)